classes/Http/Controller/ForgotController.php

```diff
@@ -168,7 +166,9 @@ public function processResetAction(Request $request)
             $form->get('user_id')->setData($userId);
             $form->get('reset_code')->setData($resetCode);
 
-            return $this->render('user/reset_password.twig', ['form' => $form->createView()]);
```

Comment:
While at it, I figured it would make sense to streamline how we pass data to the templates - this is an example.